eng/pipelines/common/templates/pipeline-with-resources.yml

```diff
@@ -5,75 +5,75 @@ parameters:
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-cross-arm64-alpine
       env:
         ROOTFS_DIR: /crossrootfs/arm64
     # This container contains all required toolsets to build for Android and for Linux with bionic libc.
 
     - container: Linux_bionic
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-android-20220808192756-8fcaabc
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-android
 
     - container: Linux_x64
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-20210714125435-9b5bbc2
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
 
     - container: Linux_x86
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-x86-linux-20211022152824-f853169
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-x86-linux
       env:
         ROOTFS_DIR: /crossrootfs/x86
 
     - container: SourceBuild_Linux_x64
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-source-build-20210714125450-5d87b80
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-source-build
 
     - container: Linux_s390x
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-s390x-20201102145728-d6e0352
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-s390x
       env:
         ROOTFS_DIR: /crossrootfs/s390x
 
     - container: Linux_ppc64le
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-ppc64le-20220531132048-b9de666
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-ppc64le
       env:
         ROOTFS_DIR: /crossrootfs/ppc64le
 
     - container: Browser_wasm
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-webassembly-20220531132048-00a561c
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-webassembly
 
     - container: FreeBSD_x64
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-freebsd-12-20210917001307-f13d79e
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-freebsd-12
       env:
         ROOTFS_DIR: /crossrootfs/x64
 
     - container: Tizen_armel
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-armel-tizen-20210719212651-8b02f56
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-armel-tizen
       env:
         ROOTFS_DIR: /crossrootfs/armel
 
     - container: debpkg
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-debpkg-20220504035737-cfdd435
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-debpkg
 
     - container: rpmpkg
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-8-rpmpkg-20210714125410-daa5116
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-rpmpkg
```

Comment:
I think we upgraded this due to the Ruby scripts that we run to make packaging not being available on centos 7. Moving backwards might break the official build.

Reply:
good catch. I'm not sure if this was intentional @michellemcdaniel ?

Reply:
I fixed this in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/718. Ruby is available on the centos 7 containers. Centos 8 is eol and we no longer have containers for it.

Reply:
right. I forget about the 8 story.